src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java

```diff
@@ -129,9 +129,17 @@ public void testAdd() {
 
     values = client.range("seriesAdd", 500L, 4600L, Aggregation.STD_P, 2000L);
     Assert.assertEquals(3, values.length);
-    Assert.assertEquals(1.05, values[0].getValue(), 0.0000001);
+    Assert.assertEquals(0L, values[0].getTime());
```

Comment:
@filipecosta90 please change the test such that one of the ranges values will not be 0.0